ios/FluentUI/Tooltip/TooltipPositionController.swift

```diff
@@ -76,12 +76,30 @@ class TooltipPositionController {
 
     private let arrowMargin: CGFloat
     private let arrowWidth: CGFloat
+    private let arrowHeight: CGFloat
+    private let totalPaddingVertical: CGFloat
+    private let paddingHorizontal: CGFloat
+    private let maximumWidth: CGFloat
+    private let messageLabelFont: UIFont
+    private let titleLabelFont: UIFont
     private let boundingRect: CGRect
     private let offset: CGPoint
 
     private var tooltipSize: CGSize = .zero
-
-    init(anchorView: UIView, message: String, boundingRect: CGRect, preferredArrowDirection: Tooltip.ArrowDirection, offset: CGPoint, arrowMargin: CGFloat, arrowWidth: CGFloat) {
+    init(anchorView: UIView,
+         message: String,
+         title: String? = nil,
```

Comment:
because tooltippositioncontroller is a private clss anyway, can't we just pass in the tooltiptokenset rather than adding tons of new param? any time we update a token we have to update multiple different functions?

Reply:
then we can get rid of bunch of private let too?